### PR TITLE
Remove new lines in FlashEdison.json

### PR DESCRIPTION
**Pull request by zvin**

### CODE REVIEW

**Comment by floion** [comment]
@zvin can we have also a CHANEGLOG entry as part of this PR with a short description for it please?

**Comment by zvin** [comment]
@floion done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Change log
 -----------
 
+# v2.7.2+rev3
+## (2017-10-18)
+
+* Fix the FlashEdison.json file: remove unescaped newlines in strings [Alexis]
```
Comment (floion) [inline CHANGELOG.md]: We don't need the 3 extra lines from above

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
```
Comment (floion) [inline CHANGELOG.md]: We don't need the extra line below

File: CHANGELOG.md (hunk 1 above)
Comment (zvin) [reply CHANGELOG.md]: Removed

File: CHANGELOG.md (hunk 2 above)
Comment (zvin) [reply CHANGELOG.md]: Removed

**Comment by floion** [comment]
@resin-jenkins retest this please

**Comment by floion** [comment]
Thanks Alexis

**Pull request merged**